src/api/general-services/work-submit/create-worker-k8s.js

```diff
@@ -63,27 +64,54 @@ const createWorkerResources = async (service) => {
 
   // Attempt to deploy the worker.
   try {
-    const params = `upgrade worker-${workerHash} chart-instance/ --namespace ${cfg.namespace} -f ${name} --install --wait -o json`.split(' ');
+    const params = `upgrade worker-${workerHash} chart-instance/ --namespace ${cfg.namespace} -f ${name} --install --atomic -o json`.split(' ');
 
     let { stdout: release } = await execFile(HELM_BINARY, params);
     release = JSON.parse(release);
 
     logger.log(`Worker instance ${release.name} successfully created.`);
   } catch (error) {
+    const params = `history worker-${workerHash} --namespace ${cfg.namespace}`;
+    logger.log(`helm update failed. Calling "helm ${params}"...`);
+    const history = await execFile(HELM_BINARY, params.split(' '));
+    logger.log(history.stdout);
+    logger.log(`If the chart is stuck updating, you may need to call "helm ${params.replace('history', 'rollback')}"`);
```

Comment:
What does this command we are telling the user to run do? Is it the manual intervention that was mentioned in the ticket?

Reply:
* We are not telling users, but telling ourselves when looking at the lens logs :-D
* Yes, this is intended to make simpler the manual intervention if the need arises